### ISSUE TITLE
[no-restricted-disable] should `rule: off` be reported as error?

### ISSUE THREAD

**Issue by bogdan-zaharia-hs** [mysticatea/eslint-plugin-eslint-comments#48]
When enabling the `no-restricted-disable` rule, I guess both the following statements should be reported as errors:
```js
/* eslint-disable max-lines */ 
/* eslint max-lines: off */
```
But only the first one is reported. Shouldn't the second statement be an error as well?

Thanks!

**Comment by MichaelDeBoey** [comment]
Hi @bogdan-zaharia-hs!

Since this repo is unmaintained, you might want to re-open this issue in the @eslint-community fork https://github.com/eslint-community/eslint-plugin-eslint-comments

For more info about why we created this organization, you can read https://eslint.org/blog/2023/03/announcing-eslint-community-org
